app/models/concerns/hostext/search.rb

```diff
@@ -11,27 +11,27 @@ module Search
 
       scoped_search :on => :name,          :complete_value => true, :default_order => true
       scoped_search :on => :last_report,   :complete_value => true, :only_explicit => true
-      scoped_search :on => :comment,       :complete_value => true
+      scoped_search :on => :comment,       :complete_value => true, :only_explicit => true
```

Comment:
I removed the comment because it is a text field, so searching it may be slower then searching a string field.